app/views/projects/_form.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= form_with(model: @project, local: true) do |form| %>
+  <%= render 'layouts/errors', object: @project %>
+
+  <div class="form-group">
+    <%= form.label :title %>
+    <%= form.text_field :title, id: :project_title, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :description %>
+    <%= form.text_field :description, id: :project_description, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :cost %>
+    <%= form.number_field :cost, id: :project_cost, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :start_date %>
+    <%= form.date_field :start_date, id: :project_start_date, class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :status %>
+    <%= form.select :status, options_for_select(Project.statuses.map { |key, value| [key.humanize, key] }) %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :client %>
+    <%= form.collection_select :client_id, @all_clients, :id, :name, include_blank: "Select client" %>
```

Comment:
you can use select here, if you go with something like pluck(:name, :id) in your controller as only these two are needed here, no need for collection select.